multigeiger/ble.cpp

```diff
@@ -0,0 +1,141 @@
+// Project: Simple Multi-Geiger
+// Modul: BLE transmission of CPM data
```

Comment:
"Module" (or remove).

also I would not specify what data is transferred, that can change any time and you don't want to have to update that line all the time.

Reply:
ok, will change to Multigeiger data